controller/eks-cluster-config-handler.go

```diff
@@ -359,7 +359,7 @@ func validateUpdate(config *eksv1.EKSClusterConfig) error {
 			"must be equal to or one minor version lower than the cluster kubernetes version", aws.ToString(config.Spec.KubernetesVersion), aws.ToString(ng.Version)))
 	}
 	if len(errs) != 0 {
-		return fmt.Errorf(strings.Join(errs, ";"))
+		return fmt.Errorf("%s", strings.Join(errs, ";"))
```

Comment:
If formatting is not required, perhaps we can use this instead, wdyt? This would require us to also import "errors" package from standard library.
```suggestion
		return errors.New(strings.Join(errs, ";"))
```